.github/workflows/Jaseci_core_build.yml

```diff
@@ -0,0 +1,33 @@
+name: Jaseci_core - Build and Test
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python 3.10
+      uses: actions/setup-python@v2
+      with:
+        python-version: "3.10"
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install black flake8==3.7.9 isort
+        
+    - name: Install Jaseci
+      run: |
+        source jaseci_core/install.sh
```

Comment:
lets directly run the pip install command here instead of using bash scripts.

Reply:
done